modules/features/wim_agenda/wim_agenda.info

```diff
@@ -3,13 +3,15 @@ description = Keeps Agenda feature
 core = 7.x
 package = WIM
 version = 7.x-1.0
+dependencies[] = content_reminder
```

Comment:
@vladnosykov Please remove this dependency. Content Reminder could be disabled by default so module should not depend on it.